src/Renderer/Camera.js

```diff
@@ -4,24 +4,13 @@
  * Description: La camera scene, interface avec la camera du 3DEngine.
  */
 
-/* global Float64Array*/
-
 import * as THREE from 'three';
 
 function Camera(width, height, options = {}) {
     this.ratio = width / height;
 
     this.camera3D = options.camera ? options.camera : new THREE.PerspectiveCamera(30, this.ratio);
 
-
-    if (!options.camera) {
-        // do not modify a camera we don't own fully
-        // /!\ WARNING Matrix JS are in Float32Array
-        this.camera3D.matrixWorld.elements = new Float64Array(16);
-        this.camera3D.matrixAutoUpdate = false;
-        this.camera3D.rotationAutoUpdate = false;
```

Comment:
No.
The first line is useless now that three.js matrices use double.
The 2nd/3rd lines are optimization that bring no performance gain but bring troubles: the user have to remember to call `camera.updateMatrix`, otherwise the camera will never move.